components/navbar/navbar.snip.html

```diff
@@ -19,7 +19,7 @@
 {{#navbar}}
   <!-- Start Navbar -->
   <header class="ampstart-headerbar fixed flex justify-start items-center top-0 left-0 right-0 pl2 pr4">
-    <div role="button" aria-label="open sidebar" on="tap:{{sidebar-id}}.toggle" tabindex="0" class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2">{{#svg-trigger}}{{{.}}}{{/svg-trigger}}{{^svg-trigger}}☰{{/svg-trigger}}
+    <div role="button" aria-label="open sidebar" on="tap:{{sidebar-id}}.toggle" tabindex="0" class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2 {{trigger-layout-classes}}">{{#svg-trigger}}{{{.}}}{{/svg-trigger}}{{^svg-trigger}}☰{{/svg-trigger}}
```

Comment:
Trigger layout classes is added directly to the trigger button, so it doesn't make sense to just call it layout classes right now. When the sidebar data is refactored as above we can make the change.